docs/class-definitions.md

```diff
@@ -74,3 +83,27 @@ This could be the local client or other remote clients which are connected to th
 | isConnected | bool                              |
 | profileData | Object                            |
 | lastEvent   | {name: string, timestamp: number} |
+
+
+### Locations
+
+Handles the tracking of member locations within a space.
+
+#### set(location)
+Set your current location. Location can be any JSON-serializable object. Fires a [locationUpdate](#locationupdate) event for all connected clients in this space.
+
+| Property | Type |
+|----------|------|
+| location | Any  |
+
+#### on(event, callback)
+Used for subscribing to location specific updates. Currently, only one event is supported:
+
+##### locationUpdate
+Fires when a member updates their location. The argument supplied to the event listener is an object with the following fields:
+
+| Property         | Type                        |
+|------------------|-----------------------------|
+| member           | [SpaceMember](#spacemember) |
+| currentLocation  | Any                         |
+| previousLocation | Any &#124; undefined        |
```

Comment:
Should the types for locations just both be `any`? i.e. a member might not have any current location. We also set the default for location to `null`, not `undefined`.

Reply:
Fixed https://github.com/ably-labs/spaces/pull/41/commits/ec228813f179b7068bb7b649a79fcaa81465fcc0